### PR TITLE
add migration for staking v10

### PR DESCRIPTION
**Pull request by kianenigma**
loose companion of https://github.com/paritytech/substrate/pull/11823

### CODE REVIEW

**Review by andresilva — APPROVED** [review]

File: runtime/kusama/src/lib.rs
Comment (ggwpez) [inline runtime/kusama/src/lib.rs:1607]:
```suggestion
		pallet_staking::migrations::v10::MigrateToV10<Runtime>,
```

Can this be removed now? Same for Westend.

**Review by ggwpez — APPROVED** [review]

File: runtime/kusama/src/lib.rs
Comment (kianenigma) [reply runtime/kusama/src/lib.rs:1607]: yeah, but I rather leave that to the release team. Ideally they should have a process around it. All migrations that I write have a guard and will not be re-executed.

**Review by bkchr — APPROVED** [review]

**Comment by kianenigma** [comment]
bot merge

**Comment by kianenigma** [comment]
bot merge

**Pull request merged**
